elixir/apps/web/lib/web/live/actors/show.ex

```diff
@@ -573,7 +573,7 @@ defmodule Web.Actors.Show do
 
     socket =
       socket
-      |> put_flash(:info, "Welcome email sent to #{identity.provider_identifier}")
+      |> put_flash(:info, "Welcome email sent to #{get_identity_email(identity)}")
```

Comment:
@Intuinewin Thank you! That's a good catch. Can you please add a test to make sure we don't have regression in the future?

Reply:
Done